mzbench/scenarios/fan_in.bdl

```diff
@@ -2,56 +2,71 @@
 
 #######
 # Scenario:
-# A single subscriber reading from "<MF_CHANNEL>/#" topic
-# 10000 publisher publishing to exclusive topic "<MF_CHANNEL>/<client_id>"
+# A single subscriber reading from "channels/<MF_CHANNEL_ID>/messages/#" topic
+# 10000 publisher publishing to exclusive topic "channels/<MF_CHANNEL_ID>/messages/<client_id>"
 # Overall msg rate: 10 000 msg/s
 # Message size: 100 random bytes
 # Running time: 5 min
 # QoS level: 2
-# NOTE: Update values in test, according to your values
+# NOTE: Update values in defaults or set environment variables according to your values
 # <MF_MQTT_ENPOINT> = IP/domain of your MQTT endpoint
 # <MF_THING_1_ID> = First thing ID of pre-provisioned Mainflux thing
-# <MF_THING_1_key> = First thing key of pre-provisioned Mainflux thing
+# <MF_THING_1_KEY> = First thing key of pre-provisioned Mainflux thing
 # <MF_THING_2_ID> = Second thing ID of pre-provisioned Mainflux thing
-# <MF_THING_2_key> = Second thing key of pre-provisioned Mainflux thing.
-# <MF_CHANNEL> = pre-provisioned channel where thing is connected
+# <MF_THING_2_KEY> = Second thing key of pre-provisioned Mainflux thing.
+# <MF_CHANNEL_ID> = pre-provisioned channel id where things are connected
 #######
 
+defaults(
+        "PUB_NUM" = 10000,
+        "PUB_TIME" = 5,
+        "PUB_RATE" = 1,
+        "MSG_SIZE" = 100,
+        "QOS" = 2,
+        "MF_MQTT_ENDPOINT" = "127.0.0.1",
+        "MF_MQTT_PORT" = 1883,
+        "MF_THING_1_ID" = "<MF_THING_1_ID>",
```

Comment:
These defaults (for things) do not have much sense, better to remove them. If left undefined, maybe they will rise an error in test, rather than reporting connection failure because they were malconfigured.

Reply:
[Here](https://satori-com.github.io/mzbench/scenarios/spec/#var):
```text
If you refer to an undefined variable, the benchmark crashes. You can avoid this by setting a default value for the variable, see defaults top-level directive.
```
But actually - because thing vars must be provided by the user - we **want** test to crash, so that we can see this type of error (rather than connection refused).

Reply:
Removed default values for mainflux pre-provisioned things and channel